### DISCUSSION TITLE
Can't use invalidate_session in security.yaml

### DISCUSSION THREAD

**Discussion by karaka200** [symfony/symfony#43907] [Q&A] [answered]
I add [invalidate_session](https://symfony.com/doc/current/reference/configuration/security.html#invalidate-session) to my `security.yaml` config file. When open login page then show this error:

`Unrecognized option "invalidate_session" under "security.firewalls.secured_area.form_login". Available options are "always_use_default_target_path", "check_path", "csrf_parameter", "csrf_token_generator", "csrf_token_id", "default_target_path", "enable_csrf", "failure_forward", "failure_handler", "failure_path", "failure_path_parameter", "login_path", "password_parameter", "post_only", "provider", "remember_me", "require_previous_session", "success_handler", "target_path_parameter", "use_forward", "use_referer", "username_parameter".`

How can I fix this problem?

**Comment by BahmanMehrdad** [comment, marked as answer]
You are using it in the wrong path. You must put in the following path:

```
security:
    firewalls:
        secured_area:
            logout:
                invalidate_session: ...
```
